packages/expo-firebase-analytics/src/ExpoFirebaseAnalytics.ts

```diff
@@ -31,10 +35,22 @@ function callAnalyticsModule(funcName: string, ...args) {
   // In that case we show a warning and log the analytics events to the console.
   // The user can disable these by calling `setUnavailabilityLogging(false)`.
   if (DEFAULT_APP_NAME !== '[DEFAULT]') {
+    const defaultWebOptions = getDefaultWebOptions();
+    if (defaultWebOptions && !pureJSAnalyticsTracker) {
+      pureJSAnalyticsTracker = new FirebaseAnalyticsJS(defaultWebOptions, {
+        clientId: Constants.sessionId,
+        strictNativeEmulation: true,
+        appName: Constants.manifest?.name || 'Expo Client',
+        appVersion: Constants.nativeAppVersion || undefined,
+      });
+    }
+    if (pureJSAnalyticsTracker) {
+      return pureJSAnalyticsTracker[funcName].call(pureJSAnalyticsTracker, ...args);
+    }
     if (isUnavailabilityLoggingEnabled) {
       if (!isUnavailabilityWarningLogged) {
         console.warn(
-          'Firebase Analytics is not available in the Expo client. To test Firebase Analytics create a stand-alone build or custom client. To suppress this warning use `setUnavailabilityLogging(false)`.'
+          `Firebase Analytics is not available in the Expo client. See "https://docs.expo.io/versions/unversioned/sdk/firebase-analytics#expo_client_limitations_configuration" on more information on setting up Firebase Analytics with the standard Expo Client.`
```

Comment:
```suggestion
          `Firebase Analytics is not available in the Expo client. See "https://docs.expo.io/versions/latest/sdk/firebase-analytics" on more information on setting up Firebase Analytics with the standard Expo Client.`
```

- link to latest rather than unversioned
- leave out anchor as those get stale pretty quick and we don't have any automated validation for links from code